client/packages/common/src/ui/components/navigation/AppNavLink/AppNavLink.tsx

```diff
@@ -72,7 +72,7 @@ export interface AppNavLinkProps {
   text?: string;
   to: string;
   visible?: boolean;
-  onClick?: () => void;
+  onClick?: React.MouseEventHandler<HTMLAnchorElement>;
```

Comment:
have had to add this so that I can cancel an onClick event and prevent navigation when the link is js only